src/frontends/paddle/include/openvino/frontend/paddle/node_context.hpp

```diff
@@ -19,7 +19,7 @@ using NamedInputs = std::map<InPortName, OutputVector>;
 
 /// Keep necessary data for a single node in the original FW graph to facilitate
 /// conversion process in the rules code.
-class NodeContext : public ov::frontend::NodeContext {
+class PADDLE_API NodeContext : public ov::frontend::NodeContext {
```

Comment:
Do we really need to expose this externally? Ideally, think - no

Reply:
should we move it to `src` folder then?

Reply:
Yes in this case we need to remove this API from public API